squire_lib/src/tournament.rs

```diff
@@ -1269,6 +1273,62 @@ mod tests {
             .assume_pair();
     }
 
+    #[test]
+    fn create_round_test() {
+        let mut tourn =
+            Tournament::from_preset("Test".into(), TournamentPreset::Swiss, "Test".into());
```

Comment:
There should be a method in `squire_tests` to get a "default" seed.

Reply:
This can't be used directly from a unit test in `squire_lib`. I think it's trying to compile `squire_lib` twice -- so that it can compile `squire_tests` as a dev-dependency, and the crate itself with `cfg(test)` -- and so it ends up with two different versions of the `TournamentSeed` struct. The seed can't be used destructured either, because the `TournamentPreset` type isn't standard, so it can't be used from outside `squire_tests`. It could work if you re-export all the types used in `squire_tests`, but I'm not sure if that's necessary.

Reply:
Ah, ya. That is a problem. That would create a circular dependency. But, we don't need to access any private fields/methods for this test, so it should probably be an integration test anyway. Any calls to `create_round` should be replaced with create round tournament operations.